srcpkgs/devilution/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'devilutionx'
+pkgname=devilutionx
+version=1.0.0
+revision=1
+wrksrc="devilutionX-${version}"
```

Comment:
Then why are you naming the pkg `devilutionx` instead of `devilutionX`?

Also, the pkgdir is incorrect.

Reply:
I am naming pkg by convention, Fedora and [Debian](http://fuzon.co.uk/meveric/pool/main/d/devilutionx/) packages are devilutionx, not devilutionX.

The pkgdir is correct, because I'm using devilutionx, instead of devilutionX.

Reply:
Don't care what their packages are named. We need a better reason than that.

Reply:
Named for devilutionX.